iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -439,6 +443,97 @@ TEST_F(PortManager_test, ConditionVariablesDestroy)
     }
 }
 
+TEST_F(PortManager_test, AcquiringMaximumNumberOfEventVariablesWorks)
+{
+    std::string process = "BuddyHolly";
+
+    for (unsigned int i = 0U; i < iox::MAX_NUMBER_OF_EVENT_VARIABLES; i++)
+    {
+        iox::ProcessName_t newProcessName(iox::cxx::TruncateToCapacity, process + std::to_string(i));
+        auto eventVariableDataResult = m_portManager->acquireEventVariableData(newProcessName);
+        EXPECT_THAT(eventVariableDataResult.has_error(), Eq(false));
+    }
+}
+
+TEST_F(PortManager_test, AcquiringOneMoreThanMaximumNumberOfEventVariableFails)
+{
+    std::string process = "BuddyHollysBrille";
```

Comment:
English please :)

Reply:
Never, since this has to be an arbitrary random string and we would like to make sure that the german language does not crash iceoryx!

Reply:
But then this must be called `BüddyHöllysBrille`